tmp/build/patch-prometheus.sh

```diff
@@ -49,16 +49,8 @@ function prometheus_patch_deployment() {
 \1containers:\2/' \
   ${HELM_DIR}/istio/charts/prometheus/templates/deployment.yaml
 
-  if [[ "${COMMUNITY,,}" == "true" ]]; then
-    sed -i -r -e 's/image:(.*)prometheus:/image:\1prometheus-ubi8:/' ${HELM_DIR}/istio/charts/prometheus/templates/deployment.yaml
-  else
-    sed -i -r -e 's/image:(.*)prometheus:/image:\1prometheus-rhel8:/' ${HELM_DIR}/istio/charts/prometheus/templates/deployment.yaml
-  fi
-
-  sed -i -e '/image.*prometheus/,/args:/ {
-      /args:/ a\
-            - --storage.tsdb.path=/prometheus
-    }' ${HELM_DIR}/istio/charts/prometheus/templates/deployment.yaml
+  sed -i -r -e 's/image:(.*)prometheus:/image:\1{{ \.Values\.image }}:/' ${HELM_DIR}/istio/charts/prometheus/templates/deployment.yaml
+	sed "/storage.tsdb.retention.*/a\ \ \ \ \ \ \ \ \ \ \ \ - \'--storage.tsdb.path=/prometheus\'" deployment.yaml
```

Comment:
Path to deployment.yaml is incorrect here. I've fixed this in https://github.com/Maistra/istio-operator/pull/116